src/main/java/com/programmers/calculator/engine/calculation/CalculatePostfix.java

```diff
@@ -0,0 +1,41 @@
+package com.programmers.calculator.engine.calculation;
+
+import java.util.List;
+import java.util.Stack;
+
+public class CalculatePostfix implements Calculate {
+    private final String NUMBER = "^[0-9]*$";
+    static Stack<Double> stack = new Stack<>();
+
+
+    @Override
+    public Double calculate(List<String> postfix) {
+        for (String token : postfix) {
+            isNumber(token);
+        }
+        return stack.pop();
+    }
+
+
+    //이렇게 세세하게 나누는 방법이 좋은건지 궁금합니다.
+    void isNumber(String token) {
+        if (token.matches(NUMBER)) {
+            pushStack(token);
+        } else {
+            calculateStack(token);
+        }
+    }
+
+    void pushStack(String token) {
+        stack.push(Double.valueOf(token));
+    }
+
+    void calculateStack(String token) {
+        double number1, number2;
+        number2 = stack.pop();
+        number1 = stack.pop();
```

Comment:
```suggestion
        double number2 = stack.pop();
        double number1 = stack.pop();
```
선언해놓고 쓰지마세요! 항상 null이 발생하지않도록 코드를 작성해주세요.